### PR TITLE
Adding sign-off action to shared flows

### PR DESCRIPTION
**Pull request by matt-smith-kr**
Adding sign-off action to shared flows.

### CODE REVIEW

File: .github/workflows/qa.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Check for Sign Off
+
+on:
+  workflow_call:
+    inputs:
```
Comment (dave-kr) [inline .github/workflows/qa.yml]: I think `inputs` here should actually be `secrets` to protect the token.

File: .github/workflows/qa.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+jobs:
+  sign-off-check:
+    name: Check sign-off
+    runs-on: ubuntu-latest
+    steps:
+      - name: sign off
+        uses: krogertechnology/sign-off@v1.0.1
+        with:
+          github-token: ${{ inputs.github-token }}
```
Comment (dave-kr) [inline .github/workflows/qa.yml]: This would become `secrets.github-token`

**Review by dave-kr — CHANGES_REQUESTED** [review]

File: .github/workflows/qa.yml (hunk 1 above)
Comment (matt-smith-kr) [reply .github/workflows/qa.yml]: That's a good point. I've made the update.

File: .github/workflows/qa.yml (hunk 2 above)
Comment (matt-smith-kr) [reply .github/workflows/qa.yml]: Updated

**Review by dave-kr — APPROVED** [review]

**Pull request merged**